.pre-commit-config.yaml

```diff
@@ -2,21 +2,19 @@
 # "Version control integration" in README.md.
 exclude: ^(blib2to3/|profiling/|tests/data/)
 repos:
--   repo: local
+  - repo: https://github.com/psf/black
+    rev: 19.3b0
     hooks:
-    - id: black
-      name: black
-      language: system
-      entry: black
-      types: [python]
-    - id: flake8
-      name: flake8
-      language: system
-      entry: flake8
-      types: [python]
-    - id: mypy
-      name: mypy
-      language: system
-      entry: mypy
-      types: [python]
-      exclude: ^docs/conf.py
+      - id: black
```

Comment:
at least for black, I believe the intention was to dogfood black itself -- the others I agree 👍

Reply:
We also have unit tests that run Black on itself, so this check would likely conflict with those tests if a PR changes formatting.

Reply:
I think the version pre-commit installs is _only_ available for pre-commit, so wouldn't affect those unit tests.

Anyway, I've reverted this bit so Black dogfoods for pre-commit as well.